tools/sumolib/version.py

```diff
@@ -25,15 +25,22 @@
 from __future__ import print_function
 
 import subprocess
+import os
 from os.path import dirname, exists, join
 
 UNKNOWN_REVISION = "UNKNOWN"
 GITDIR = join(dirname(__file__), '..', '..', '.git')
+verbose = False  # set to True if you want some more debug statements
 
 
 def _findVersion():
+    if verbose:
+      print('gitdir1 = %s' % GITDIR)
+      print('pwd1 = %s'    % os.getcwd())
     # try to find the version in the config.h
     versionFile = join(dirname(__file__), '..', '..', 'include', 'version.h')
+    if not exists(versionFile):
+      versionFile = join('src', 'version.h')
```

Comment:
There should be no version.h in src anymore. This is an artifact created only by "in tree" builds which are either using autotools or a discouraged cmake configuration.